overrides/Inventory/ui/inventoryScreen.ui

```diff
@@ -0,0 +1,219 @@
+{
+  "type": "ShopScreen",
```

Comment:
I wonder if this can live in `Economy` as well? Or does it differ from the one we have in GD? If so, in how far?